schemas/api_schema.js

```diff
@@ -63,6 +63,7 @@ type Session {
     platform_media_id: String!
     img_source: String
     resession_id: ID
+    active: Boolean!
```

Comment:
the usage of this bit is not very clear, please explain what it suppose to represent

Reply:
the linked issue will give you the answers